lib/rubocop/cop/rails/date.rb

```diff
@@ -109,6 +109,16 @@ def safe_chain?(chain)
           (chain & bad_methods).empty? || !(chain & good_methods).empty?
         end
 
+        def safe_to_time?(node)
+          receiver, _method_name, *args = *node
+          if receiver.str_type?
+            zone_regexp = /[\+-]{1}[\d:]+$/
+            receiver.str_content.match(zone_regexp)
+          else
+            args.length == 1
```

Comment:
`args.length == 1` and `args.one?` are not equal.

http://ruby-doc.org/core-1.9.3/Enumerable.html#method-i-one-3F

For example

```
[1] pry(main)> [].one?
=> false
[2] pry(main)> [1].one?
=> true
[3] pry(main)> [false].one?
=> false
[4] pry(main)> [1, nil, nil].one?
=> true
[5] pry(main)> 
```